Sources/AppStoreConnectCLI/Commands/TestFlight/Apps/ReadAppCommand.swift

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import ArgumentParser
+import Foundation
+
+struct ReadAppCommand: CommonParsableCommand {
+
+    static var configuration = CommandConfiguration(
+        commandName: "read",
+        abstract: "Find and read app info"
+    )
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Argument(
+        help: ArgumentHelp(
+            "Filter by app AppStore ID. eg. 432156789 or app bundle identifier. eg. com.example.App",
+            discussion: "Please input either app id or bundle Id",
+            valueName: "app-id / bundle-id"
+        )
+    ) var appIdOrBundleId: String
+
+    enum CommandArgument {
```

Comment:
As for naming maybe something as simple as: `Identifier`

and the above can also just be an `identifier` where it's one of two kinds of identifier (id)